.github/workflows/ci.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.7","3.8","3.9", "3.10", "3.11"]
```

Comment:
why can't we have nice things?

Reply:
the usual reasons - [numpy](https://github.com/Beauhurst/pyap/actions/runs/4144442570/jobs/7167492709)

although I think that is only for pandas, and that is maybe not neccesary (only used in the tests)

Reply:
46a60c3 brings 3.11 back, have replaced pandas with csvreader in the tests

Reply:
does this mean we can't install pandas and this package together ?

Reply:
I think the problem is that having it support 3.7 limits which version of pandas you can install

Reply:
ah, so as long as the versioning is not tied together, we should be fine